### ISSUE TITLE
Links to command line argument parsing

### ISSUE THREAD

**Issue by jsmarsch** [dotnet/docs#27339]
I know that .Net Core added a really cool command line argument parser, but when I search the documentation, all I can find is examples how how to use the args array and parse it manually.  Google doesn't help because it prioritizes old command line parser nuget packages. there should be a link on this page to the articles on using the command line parser.  It's such a shame, -- I'm having such a hard time finding the docs, I'll probably fall back to the 3rd party package that I used to use in the pre-core days.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cde79d6e-21ab-c158-8f41-1821ad920628
* Version Independent ID: 69a297cb-677e-2b05-0187-0d0ec0890b00
* Content: [How to display command-line arguments](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/tutorials/how-to-display-command-line-arguments)
* Content Source: [docs/csharp/fundamentals/tutorials/how-to-display-command-line-arguments.md](https://github.com/dotnet/docs/blob/main/docs/csharp/fundamentals/tutorials/how-to-display-command-line-arguments.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by gfoidl** [comment]
You mean https://github.com/dotnet/command-line-api?

**Comment by BillWagner** [comment]
Adding @KathleenDollard @jonsequitur 

Which library should we link to from these articles?

**Comment by jsmarsch** [comment]
Hello all:

Perhaps the one that Gunther mentioned. Unfortunately, I’ve only recently
had the opportunity to move from “Classic” .Net Framework to . Net/formerly
core. So, I know there is some sort of built-in command line parse
capability — The new (to me) configuration system can even  merge settings
from json config files, environment variables, and command line args.
There’s even a command line package that’s pulled in when you add the
Microsoft.Extensions.Hosting package to the console app. I was looking to
figure out how supply command line args in a way that works with the config
system, but I never did find the right docs. I had to keep moving though,
so I pulled in a 3rd-party parser.
